net/sync/gpoddernet/src/main/java/de/danoeh/antennapod/net/sync/gpoddernet/mapper/ResponseMapper.java

```diff
@@ -0,0 +1,64 @@
+package de.danoeh.antennapod.net.sync.gpoddernet.mapper;
+
+import androidx.annotation.NonNull;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+import de.danoeh.antennapod.net.sync.model.EpisodeAction;
+import de.danoeh.antennapod.net.sync.model.EpisodeActionChanges;
+import de.danoeh.antennapod.net.sync.model.SubscriptionChanges;
+
+public class ResponseMapper {
+    public static String convertStreamToString(java.io.InputStream is) {
```

Comment:
You can also use `IOUtils.toString` instead - then we don't need to add this function again.